pkg/kritis/secrets/secrets_test.go

```diff
@@ -81,3 +88,38 @@ func getTestSecret(namespace string, name string) (*v1.Secret, error) {
 	}
 	return nil, fmt.Errorf("Secret %s not found", name)
 }
+
+func createKeys(name string) (string, string) {
+	// Create a new pair of key
+	var key *openpgp.Entity
+	key, err := openpgp.NewEntity(name, "test", fmt.Sprintf("%s@grafeas.com", name), nil)
+	if err != nil {
+		glog.Errorf("Unexpected error: %v", err)
```

Comment:
Fatalf might be better here. Either way, this function should exit rather than continue processing.

Reply:
good point, done